cloudformation/template.yml

```diff
@@ -5,44 +5,55 @@ Metadata:
     ParameterGroups:
       -
```

Comment:
Everything in the `ParameterGroups` object relates to reordering and relabelling parameters in the interface.